plugin.jest.js

```diff
@@ -136,14 +137,55 @@ describe('CspHtmlWebpackPlugin', () => {
         const expected =
           "base-uri 'self';" +
           " object-src 'none';" +
-          " script-src 'unsafe-inline' 'self' 'unsafe-eval' 'sha256-ixjZMYNfWQWawUHioWOx2jBsTmfxucX7IlwsMt2jWvc=' 'nonce-mockedbase64string-1' 'nonce-mockedbase64string-2';" +
+          " script-src 'unsafe-inline' 'self' 'unsafe-eval' 'sha256-ixjZMYNfWQWawUHioWOx2jBsTmfxucX7IlwsMt2jWvc=' 'sha256-J7wa7HNc5Nb9SvdpRj1UEzZlXOJERU6Mw8r5DSsL1Go=' 'nonce-mockedbase64string-1' 'nonce-mockedbase64string-2';" +
           " style-src 'unsafe-inline' 'self' 'unsafe-eval' 'sha256-MqG77yUiqBo4MMVZAl09WSafnQY4Uu3cSdZPKxaf9sQ=' 'nonce-mockedbase64string-3'";
 
         expect(csps['index.html']).toEqual(expected);
         done();
       });
     });
 
+    it('inserts hashes for linked scripts and styles from the same Webpack build', (done) => {
```

Comment:
Could we add another test which checks that external scripts that are not generated by this webpack compilation don't have a hash added to the csp?

Reply:
I wasn't 100% on what you meant by this – as in external scripts linked directly from the HTML (which I _think_ should be covered by this test, since it uses `external-scripts-styles.html`)? or as in multiple webpack runs or something?